Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -2099,6 +2125,6 @@ changing "passwordID" to the correct element ID. If the value you typed into the
 	<key>pfm_unique</key>
 	<false/>
 	<key>pfm_version</key>
-	<integer>14</integer>
+	<integer>16</integer>
```

Comment:
I think the XCreds 5.6 commits were never submitted, so let's just raise the version to 15.